e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java

```diff
@@ -112,8 +112,11 @@ public static ResponseEntity<ResponseBodyEmitter> streamChatCompletion() {
 
     final Runnable consumeStream =
         () -> {
-          stream.forEach(deltaMessage -> send(emitter, deltaMessage));
-          emitter.complete();
+          try (stream) {
```

Comment:
The `Stream<String>` may `throwOnContentFilter`, without recovery. Therefore it is recommended to keep the try-with-resources.

This does not apply for `Stream<OpenAiChatCompletionDelta>` variant.